src/main/scala/mesosphere/marathon/api/akkahttp/Directives.scala

```diff
@@ -76,4 +78,20 @@ object Directives extends AuthDirectives with LeaderDirectives with AkkaDirectiv
       case Success => f(Unit)
     }
   }
+
+  /**
+    * Matches the remaining path and transforms it into an instance id or rejects if it is not a valid id.
+    */
+    class ExtractInstanceId extends Directive[Instance.Id] {
```

Comment:
We already have `PathMatchers` to match for `pathId`. Shouldn't we put this method there?